cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,51 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
-describe('Settings page', () => {
-  before(() => {
 
-  });
+import SignInPageObject from '../support/pages/signIn.pageObject';
+import homePageObject from '../support/pages/home.pageObject';
+import SettingsPageObject from '../support/pages/settings.pageObject';
 
-  beforeEach(() => {
+const updateSettings = new SettingsPageObject();
+const signInPage = new SignInPageObject();
+const homePage = new homePageObject();
 
+describe('Settings page', () => {
+  let user;
+  let bio = 'zhazam';
+  beforeEach(() => {
+    cy.task('db:clear');
+    cy.task('generateUser')
+      .then(generateUser, () => {
```

Comment:
```suggestion
      .then((generateUser) => {
```
this is the correct way, because I still cant run your tests